specification/nginx/Nginx.Management/Configurations.tsp

```diff
@@ -0,0 +1,410 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using Rest;
+using Http;
+using Azure.ResourceManager;
+
+namespace Nginx.NginxPlus;
+
+/**
+ * Represents an Nginx configuration resource.
+ */
+@parentResource(NginxDeployment)
+model Configuration is ProxyResource<NginxConfigurationResponseProperties> {
+  ...ResourceNameParameter<Configuration>;
+}
+
+/**
+ * Request model for creating or updating an Nginx configuration.
+ */
+@parentResource(NginxDeployment)
+@resource("configurations")
+model NginxConfigurationRequest
+  is ProxyResource<NginxConfigurationRequestProperties> {
+  ...ResourceNameParameter<Configuration>;
+}
+
+/**
+ * Represents a configuration file.
+ */
+model NginxConfigurationFile {
+  /**
+   * File content.
+   */
+  content?: string;
+
+  /**
+   * Virtual path of the file.
+   */
+  virtualPath?: string;
+}
+
+/**
+ * Request model for a protected configuration file.
+ */
+model NginxConfigurationProtectedFileRequest {
+  /**
+   * File content.
+   */
+  content?: string;
+
+  /**
+   * Virtual path of the file.
+   */
+  virtualPath?: string;
+
+  /**
+   * Content hash of the file.
+   */
+  contentHash?: string;
+}
+
+/**
+ * Response model for a protected configuration file.
+ */
+model NginxConfigurationProtectedFileResponse {
+  /**
+   * Virtual path of the file.
+   */
+  virtualPath?: string;
+
+  /**
+   * Content hash of the file.
+   */
+  contentHash?: string;
+}
```

Comment:
which hash algorithm is it?

Reply:
I've added to the documentation the hashing algorithm, currently it's SHA256 henceforth, won't change in the future.